CMakeLists.txt

```diff
@@ -420,14 +420,24 @@ if(MIOPEN_USE_ROCBLAS)
     set(MIOPEN_PACKAGE_REQS "${MIOPEN_PACKAGE_REQS}, rocblas")
 endif()
 
-set(CPACK_DEBIAN_PACKAGE_DEPENDS "${MIOPEN_PACKAGE_REQS}, rocm-opencl-dev")
-set(CPACK_RPM_PACKAGE_REQUIRES "${MIOPEN_PACKAGE_REQS}, rocm-opencl-devel")
-
-# Make backends explicitly conflict
 if(MIOPEN_BACKEND STREQUAL "HIP")
+    # In HIP backend, there is a posibility of HIPRTC disabled.
+    # In this case add the clang-ocl as dependency for runtime kernel compilation
+    if(NOT MIOPEN_USE_HIPRTC )
+        set(MIOPEN_PACKAGE_REQS "${MIOPEN_PACKAGE_REQS}, rocm-clang-ocl")
```

Comment:
> When HIPRTC is enabled the runtime compilation is performed through hiprtc and comgr apis. 

The above is true, however please note:
- clang-ocl is used to build OpenCL kernels. 
- HIPRTC is used to build HIP kernels. Therefore, disabling HIPRTC does not affect OpenCL build path.
- HIPRTC requires COMgr to be enabled.
- Whenever COMgr is enabled, ASM and OpenCL go via COMgr.

This means that we do not need clang-ocl when we have COMgr. This is not a bug, but let's better fix this.